### PR TITLE
Add PlayNow.app v1.0

### PR DESCRIPTION
**Pull request by ronanrodrigo**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download playnow` is error-free.
- [x] `brew cask style --fix playnow` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install playnow` worked successfully.
- [x] `brew cask uninstall playnow` worked successfully.

### CODE REVIEW

File: Casks/playnow.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'playnow' do
+  version '1.0'
+  sha256 'c082bc144ca1991bf045ee5f416a0066c549e6adb35d49edf881a58f8dbec801'
+
+  url 'https://raw.githubusercontent.com/apalancat/PlayNow/gh/PlayNow.tgz'
+  name 'PlayNow'
+  homepage 'https://raw.githubusercontent.com/apalancat/PlayNow'
```
Comment (vitorgalvao) [inline Casks/playnow.rb]: `homepage 'https://github.com/marcboquet/PlayNow'`

File: Casks/playnow.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'playnow' do
+  version '1.0'
+  sha256 'c082bc144ca1991bf045ee5f416a0066c549e6adb35d49edf881a58f8dbec801'
+
+  url 'https://raw.githubusercontent.com/apalancat/PlayNow/gh/PlayNow.tgz'
```
Comment (vitorgalvao) [inline Casks/playnow.rb]: `url "https://github.com/marcboquet/PlayNow/releases/download/v#{version}/PlayNow.tgz"` from [the releases page](https://github.com/marcboquet/PlayNow/releases).

File: Casks/playnow.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'playnow' do
+  version '1.0'
```
Comment (vitorgalvao) [inline Casks/playnow.rb]: `version '0.4'` (where are you getting `1.0` from)?

File: Casks/playnow.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/playnow.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/playnow.rb (hunk 1 above)
Comment (ronanrodrigo) [inline Casks/playnow.rb]: Done

File: Casks/playnow.rb (hunk 2 above)
Comment (ronanrodrigo) [inline Casks/playnow.rb]: Done

Comment (ronanrodrigo) [inline Casks/playnow.rb]: Done

File: Casks/playnow.rb (hunk 3 above)
Comment (ronanrodrigo) [inline Casks/playnow.rb]: I saw in a wrong place. Fixed

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**